### PR TITLE
Update answers

### PR DESCRIPTION
**Pull request by amyfain**
Check for fields before accessing. Currently hitting an error with the 'ref' field in our Stitch Integration.

### CODE REVIEW

File: tap_typeform/streams.py [hunk 1]
```diff
@@ -174,10 +174,10 @@ def sync_form(atx, form_id, start_date, end_date):
                     answer_value = answer[data_type]
 
                 answers_data_rows.append({
-                    "landing_id": row['landing_id'],
-                    "question_id": answer['field']['id'],
-                    "type": answer['field']['type'],
-                    "ref": answer['field']['ref'],
+                    "landing_id": row.get('field',{}).get('landing_id'),
```
Comment (luandy64) [inline tap_typeform/streams.py]: This doesn't need to `.get('field', {})`, it can just be `row.get('landing_id')`

Comment (amyfain) [reply tap_typeform/streams.py]: I reverted that line.

File: tap_typeform/streams.py [hunk 2]
```diff
@@ -175,9 +175,9 @@ def sync_form(atx, form_id, start_date, end_date):
 
                 answers_data_rows.append({
                     "landing_id": row['landing_id'],
```
Comment (luandy64) [inline tap_typeform/streams.py]: This is what it should look like: `"landing_id": row.get('landing_id'),`

Comment (amyfain) [reply tap_typeform/streams.py]: Perfect--made that change too!

**Pull request merged**